tiny_dnn/util/parameter_init.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
@edgarriba sure, althouh I am confused, how can I add tests for xavier I mean they are uniform right ? Fixing random seed ?